lib/tarpon/entity/offering.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Tarpon
+  module Entity
+    Package = Struct.new(:identifier, :platform_identifier)
+    class Offering
+      attr_reader :identifier, :description, :packages
+
+      def self.from(hash)
```

Comment:
Still not 100% happy with that, I stick to my suggestion of having the constructor handling this:
```ruby
def initializer(identifier:, description:, packages:, **)
  ...
end
```

Maybe I'm missing something, but I won't be picky as this method will be mostly used internally.

Reply:
Ah, I guess now I see what you mean. 
And then use it like `Offering.new(**hash)`?

Reply:
Just `Offering.new(hash)`, having `**` is just to allow passing optional fields that you really don't care about, so it won't break if RC releases a change adding new fields.